docs/src/pages/[platform]/components/stepperfield/examples/StepperFieldThemeExample.tsx

```diff
@@ -0,0 +1,37 @@
+import { StepperField, ThemeProvider } from '@aws-amplify/ui-react';
+
+const theme = {
+  name: 'stepper-theme',
+  tokens: {
+    components: {
+      stepperfield: {
+        borderColor: { value: '{colors.brand.secondary.20}' },
+        input: {
+          color: { value: '{colors.brand.primary.80}' },
+          fontSize: { value: '{fontSizes.large}' },
+        },
+        button: {
+          color: { value: '{colors.brand.primary.80}' },
+          backgroundColor: { value: '{colors.neutral.20}' },
+          _disabled: {
+            color: { value: '{colors.brand.primary.80}' },
+          },
+        },
+      },
+    },
+  },
+};
+
+export const StepperFieldThemeExample = () => {
+  return (
+    <ThemeProvider theme={theme}>
+      <StepperField
```

Comment:
Can we hide the label since it's difficult to view on dark mode?

Reply:
this label has been hidden